cfgov/cfgov/settings/test.py

```diff
@@ -3,7 +3,7 @@
 DATABASES = {
     'default': {
         'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': os.path.join(REPOSITORY_ROOT, 'db.sqlite3'),
+        'NAME': os.path.join(REPOSITORY_ROOT, 'test_db.sqlite3'),
```

Comment:
Yeah I think we can change it back. At one point we were going to use it hence the change, but it's no longer needed.